ros2_control_demo_robot/launch/rrbot_system_position_only.launch.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 ROS2-Control Development Team (2020)
+# Copyright 2021 Stogl Denis Stogl (Stogl Robotics Consulting)
```

Comment:
```suggestion
# Copyright 2021 Stogl Robotics Consulting
```
that's a lot of Stogl there!

Reply:
Actually, I didn't want to change those. Because the files are from the last year.... I will revert this.